docker/dockerfile-build-ubuntu-rock

```diff
@@ -32,9 +32,15 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-ins
     apt-get clean && \
     rm -rf /var/lib/apt/lists/*
 
+# The version of MessagePack supplied in Ubuntu 16.04 is old and won't work,
+# so we'll manually download and install the DEB files
+RUN curl -O http://ftp.us.debian.org/debian/pool/main/m/msgpack-c/libmsgpack-dev_3.0.1-3_amd64.deb \
+         -O http://ftp.us.debian.org/debian/pool/main/m/msgpack-c/libmsgpackc2_3.0.1-3_amd64.deb
```

Comment:
this should be an apt-get command. We should not download from these site directly with apt can handle the mirros for us.